constraints_plone43.txt

```diff
@@ -1,3 +1,3 @@
 -c https://dist.plone.org/release/5.1-latest/requirements.txt
-# setuptools==40.8.0
-# zc.buildout==2.13.1
+# setuptools==42.0.2
+# zc.buildout==2.13.3
```

Comment:
This are just comments and should probably go away

Reply:
Okay. I'm going to remove it.

This comment is also on bobtemplates.plone. Do you think it should be removed there as well so that packages created with it will come without it?